pkg/ui/workspaces/cluster-ui/src/api/indexDetailsApi.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+import { cockroach } from "@cockroachlabs/crdb-protobuf-client";
+import { fetchData } from "src/api";
+
+export type TableIndexStatsRequest = cockroach.server.serverpb.TableIndexStatsRequest;
+export type TableIndexStatsResponse = cockroach.server.serverpb.TableIndexStatsResponse;
+export type TableIndexStatsResponseWithKey = {
+  indexStatsResponse: TableIndexStatsResponse;
+  key: string;
+};
+
+export type ResetIndexUsageStatsRequest = cockroach.server.serverpb.ResetIndexUsageStatsRequest;
+export type ResetIndexUsageStatsResponse = cockroach.server.serverpb.ResetIndexUsageStatsResponse;
```

Comment:
Do we need to export these request & response types? I don't see them being used.